tools/check-git-version.js

```diff
@@ -0,0 +1,23 @@
+const semver = require('semver');
+const shell = require('shelljs');
+
+const GIT_MINIMUM_VERSION = '2.33.0';
+function checkGitVersion() {
+  try {
+    const regex = /[\d.]+(?=)/g;
```

Comment:
move to module scop const
```suggestion
    const regex = /\d+\.\d+\.\d+/;
```

Only match version of:

```
git version 2.33.0
git version 2.33.0.windows.2
```